code/drasil-docLang/lib/Drasil/Sections/SpecificSystemDescription.hs

```diff
@@ -141,21 +146,30 @@ generalDefinitionIntro _ = foldlSP [S "This", phrase section_,
                        
 -- | Similar to 'genDefnF', but for Data Definitions. It also uses 'EmptyS' if the ending 'Sentence' is not needed rather than an empty list.
 dataDefnF :: Sentence -> [Contents] -> Section
-dataDefnF endingSent otherContents = SRS.dataDefn
-  (dataDefinitionIntro endingSent : otherContents) []
+dataDefnF _          []            = SRS.dataDefn [dataDefnIntroNoContent] []
+dataDefnF endingSent otherContents = SRS.dataDefn (dataDefinitionIntro 
+                                        endingSent : otherContents) []
 
 -- | Creates a general Data Definition introduction. Appends the given 'Sentence' to the end.
 dataDefinitionIntro :: Sentence -> Contents
 dataDefinitionIntro closingSent = mkParagraph (foldlSent [S "This", phrase section_, 
     S "collects and defines all the", plural datum, 
     S "needed to build the", plural inModel] +:+ closingSent)
 
+dataDefnIntroNoContent :: Contents
+dataDefnIntroNoContent = mkParagraph $ S "There are no data definitions."
```

Comment:
Above comment applies here too (and below too, which I won't point out explicitly).

Reply:
Ok, I will change it!